src/aleph_vrf/exceptions.py

```diff
@@ -72,6 +72,25 @@ def __str__(self):
         )
 
 
+class PublishedHashesDoNotMatch(VrfException):
+    """
+    The random number hash received from /publish is different from the one received from /generate.
+    """
+
+    def __init__(
+        self, executor: ExecutorVRFResponse, generation_hash: str, publication_hash: str
```

Comment:
Should we use `ItemHash` from aleph-message for all these hashes ?

Reply:
These two hashes aren't `ItemHash`, are the hashes generated from the VRF.